native/SampleApps/RestAPIExplorer/RestAPIExplorer/ViewControllers/RootViewController.swift

```diff
@@ -568,13 +568,10 @@ class RootViewController: UIViewController {
         }
         
         let request = RestRequest(method: method, path: path, queryParams: queryParams)
-        RestClient.shared.send(request: request) { result in
-            switch result {
-                case .success(let response):
-                    self.handleSuccess(request: request, response: response)
-                case .failure(let error):
-                    self.handleError(request: request, error: error)
-            }
+        
+        self.view.endEditing(true)
```

Comment:
what's this change for? is the same call at 562 still needed?

Reply:
@bbirman OK, Fixed my goof.  Thanks for catching that .